libpkgconf/pkg.c

```diff
@@ -711,8 +711,29 @@ pkgconf_pkg_scan_dir(pkgconf_client_t *client, const char *path, void *data, pkg
 	pkgconf_pkg_t *outpkg = NULL;
 
 	dir = opendir(path);
-	if (dir == NULL)
+	if (dir == NULL) {
```

Comment:
Perhaps we want to check for `PKG_CONFIG_EXT` (`.pc`) here before trying to parse the file?